ingestion/src/metadata/profiler/profiler/interface/pandas/pandas_profiler_interface.py

```diff
@@ -118,10 +126,14 @@ def _(
         try:
             row = []
             for metric in metrics:
-                for df in dfs:
-                    row.append(
-                        metric().df_fn(df.astype(object).where(pd.notnull(df), None))
+                row.append(
+                    metric().df_fn(
+                        [
+                            df.astype(object).where(pd.notnull(df), None)
+                            for df in self.dfs
+                        ]
                     )
```

Comment:
why are we converting all the dataframe to objects?

Reply:
same as here: https://github.com/open-metadata/OpenMetadata/pull/10603#discussion_r1158047450
modified it